apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -1981,13 +1990,21 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
         if (isNaN(count)) {
             count = 0;
         }
-        if (!this.isUpdateMode) {
-            this.adjustPaymentBalanceDueData = this.invFormData.voucherDetails.grandTotal - this.adjustPaymentData.totalAdjustedAmount - depositAmount + this.invFormData.voucherDetails.tcsTotal + this.invFormData.voucherDetails.tdsTotal;
+        if (!depositAmount) {
```

Comment:
It will execute the if block if the `depositAmount` is 0

Reply:
yes we required in that case only